libs/ml/examples/boston_housing_distributed_learning/main.cpp

```diff
@@ -258,13 +260,20 @@ int main(int ac, char **av)
     clients[i]->SetCoordinator(coordinator);
   }
 
+  std::string homedir = getenv("HOME");
+  std::ofstream lossfile(homedir + "/Development/pysyft_distributed_w2v/results/fetch_" + std::to_string(number_of_clients) + "_SGD_" + std::to_string(float(learning_rate)) + "_" + std::to_string(seed) + "_FC3.csv", std::ofstream::out);
```

Comment:
again, instead of hard coding this in lets just set the relevant input arguments